dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseUtils.java

```diff
@@ -328,24 +330,54 @@ public static void main(String[] argv) {
                     e.printStackTrace();
                     System.exit(1);
                 }
+            } else if (argv[0].equalsIgnoreCase("update-sequences")) {
+                try (Connection connection = dataSource.getConnection()) {
+                    String dbType = getDbType(connection);
+                    String sqlfile = "org/dspace/storage/rdbms/sqlmigration/" + dbType +
+                            "/update-sequences.sql";
+                    InputStream sqlstream = DatabaseUtils.class.getClassLoader().getResourceAsStream(sqlfile);
+                    if (sqlstream != null) {
+                        StringBuilder sb = new StringBuilder();
+                        try (BufferedReader br = new BufferedReader(new InputStreamReader(sqlstream))) {
+                            for (String line = br.readLine(); line != null; line = br.readLine()) {
+                                sb.append(line);
+                            }
+                        }
```

Comment:
You should be able to replace all this fancy string parsing with just one line:
`String sqlString = IOUtils.toString(sqlstream, "UTF-8");`

That line will convert the UTF-8 InputStream into a String.

IOUtils requires an `import org.apache.commons.io.IOUtils`.

Reply:
Added commit 22cc5469a23